evm_loader/test_delete_account.py

```diff
@@ -0,0 +1,150 @@
+from solana.publickey import PublicKey
+from solana.transaction import AccountMeta, TransactionInstruction, Transaction
+from solana.rpc.types import TxOpts
+import unittest
+import base58
+from eth_account import Account as EthAccount
+
+from eth_tx_utils import make_keccak_instruction_data, make_instruction_data_from_tx
+from solana_utils import *
+
+CONTRACTS_DIR = os.environ.get("CONTRACTS_DIR", "evm_loader/")
+evm_loader_id = os.environ.get("EVM_LOADER")
+
+class EvmLoaderTestsNewAccount(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        print("\ntest_delete_account.py setUpClass")
+
+        wallet = WalletAccount(wallet_path())
+        cls.loader = EvmLoader(wallet, evm_loader_id)
+        cls.acc = wallet.get_acc()
+
+        # Create ethereum account for user account
+        cls.caller_ether = eth_keys.PrivateKey(cls.acc.secret_key()).public_key.to_canonical_address()
+        (cls.caller, cls.caller_nonce) = cls.loader.ether2program(cls.caller_ether)
+
+        if getBalance(cls.caller) == 0:
+            print("Create caller account...")
+            _ = cls.loader.createEtherAccount(cls.caller_ether)
+            print("Done\n")
+
+        print('Account:', cls.acc.public_key(), bytes(cls.acc.public_key()).hex())
+        print("Caller:", cls.caller_ether.hex(), cls.caller_nonce, "->", cls.caller,
+              "({})".format(bytes(PublicKey(cls.caller)).hex()))
+    
+    def deploy_contract(self):
+        print("deploy contract: ")
+        program_and_code = self.loader.deployChecked(
+                CONTRACTS_DIR+'SelfDestructContract.binary',
+                self.caller,
+                self.caller_ether
+            )
+        owner_contract = program_and_code[0]
+        contract_code = program_and_code[2]
+        print("contract id: ", owner_contract, solana2ether(owner_contract).hex())
+        print("code id: ", contract_code)
+        return (owner_contract, contract_code)
+
+    def make_transaction(self, owner_contract, nonce, position):
+        tx = {
+            'to': solana2ether(owner_contract),
+            'value': 1,
+            'gas': 1,
+            'gasPrice': 1,
+            'nonce': nonce,
+            'data': '183555f0',
```

Comment:
Use`abi.function_signature_to_4byte_selector` like this https://github.com/neonlabsorg/neon-evm/blob/0a7a9c44681ab180a5bc8e4c7d270ce7c83004f6/evm_loader/test_cli_emulate.py#L35